lib/interpreters/periodic_visits.rb

```diff
@@ -172,6 +172,7 @@ def generate_services(vrp)
             new_service = nil
             if !service.unavailable_visit_indices || service.unavailable_visit_indices.none?{ |unavailable_index| unavailable_index == visit_index }
               new_service = Marshal.load(Marshal.dump(service))
+              new_service.original_id = new_service.id
```

Comment:
It has to be defined in models/vrp.rb within the create method. This way, we are sure it is defined for every service or shipment.

Here, it is only defined in a periodic context